FD 1.17.0: Documentation other than the release notes (#135)
mulesoft/docs-design-center: modules/ROOT/pages/fd-connection-delete.adoc

= Delete a Connection
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords:

:imagesdir: ../assets/images

You can delete a connection that is being used by or is available to be used by a specific type of connector. For example, if the connection is available for the Database connector.

> To delete a connection, you must log into Anypoint Platform with the user ID that was used to create the connection. You cannot delete a connection that was created under a user ID that you are not logged in with.

Reviewer: ```suggestion
To delete a connection, you must log in to Anypoint Platform with the user ID that was used to create the connection. You cannot delete a connection that was created under a user ID that you are not logged in with.
```
Do we really need both sentences? They say the same thing.

> If the connection is shared because you selected the *Share this connection with my business group* checkbox when you created it, deleting the connection affect all of the connectors of that type in all of the Mule-app projects and running Mule apps in your business group.

Reviewer: Consider...:
```suggestion
If the connection is shared with you business group, deleting it affects all of the connectors of that type in all of the Mule apps and projects in your business group.
```

If a Mule-app project is using the connection, an error indicator appears on the card that represents the connector. The error indicator appears when a configuration that uses the connection is reopened, the project is reopened, a user tries to add a new configuration for the same connector type, or a user clicks *Test* in the project.

If a deployed Mule app is using the connection, the app continues to do so. If you modify the Mule app, you must switch to a different connection before you redeploy it.

> *To delete a connection:*

Reviewer: Avoid using these inline headings, per previous edit comments.
Try to separate the conceptual information from the task steps.
GLOBALs

> . Open a Mule-app project that includes a connector of the right type.

Reviewer: What is "the right type"?
```suggestion
. Open a Mule-app project that includes a connector type that you want to delete.
```

. Click the card that represents the connector type.
. At the top of the card, click *Edit*.
. Locate the connection in the list of available connections.
+
Ensure that you locate the correct connection. There is no confirmation dialog to ask you whether you really want to delete the selected connection.
. Click the three dots to the right of the connection's name and click *Delete*.
. Click *Save*.
. Click *Cancel* or *Save* to close the configuration dialog for the connector.
+

> It doesn't matter which button you click. The connection is permanently deleted when you click *Save* in the *Connection configuration* dialog.

Reviewer: Ditto prev. comment about this sentence. Cancel doesn't make sense to me.
